spec/support/load_and_extend_graph_from_file.rb

```diff
@@ -0,0 +1,7 @@
+def load_and_extend_graph_from_file(filename)
```

Comment:
Would it be better to define `load_and_extend_graph_from_file` in a namespace using a `Module`? Thoughts anyone?